packages/data/src/components/with-select/test/index.js

```diff
@@ -64,6 +64,39 @@ describe( 'withSelect', () => {
 		} );
 	} );
 
+	it( 'passes through the component context to the registry store', () => {
+		const getSelectors = jest.fn();
+
+		const genericStore = {
```

Comment:
I might call this `customStore` or something just to show the intent of this API better. If you don't provide a context we'd fall back to the _actual_ `wp-data` generic store, right?

Reply:
Updated in f77e80c